Resources/Prototypes/_RMC14/Entities/Structures/Windows/windows.yml

```diff
@@ -27,6 +27,13 @@
     price: 75
   - type: Physics
     bodyType: Static
+  - type: InteractionPopup
```

Comment:
Cooldown for this is supposed to be 10 seconds to avoid spam

https://github.com/user-attachments/assets/b7912a60-47bc-4989-8bb3-7a928bd816be